GOVERNANCE.md

```diff
@@ -1,46 +1,30 @@
-# Governance of the Deluge Community Firmware Project
+# Goal
 
-## Direction and Strategy
+Goal of the DelugeFirmware community is to produce high quality firmware application releases for users of the Deluge hardware with improvements and new features coming from our community of contributors. To be as transparent and inclusive as possible [contribution guidelines](CONTRIBUTING.md) have been defined with the intent to help reach this goal. Please also see the [Code of Conduct](CODE_OF_CONDUCT.md).
```

Comment:
I think this is an "ideal" we all share, or a good Vision, but I'm a bit skeptical as to whether or not that's actually our short-to-medium term goal right now. In my observation, and I may be way off here, our goal seems to be "getting the firmware and surrounding workflow to a stable, manageable, maintainable, and workable state by and for the community." or something like that. Interested in thoughts and whether splitting long-term vision from short-term goal would make the bar easier to reach.